tools/Generator/Xamarin.Forms.Core.json

```diff
@@ -2014,6 +2040,430 @@
+          "modelType": "Xamarin.Forms.ShellNavigationState * Fabulous.DynamicViews.AnimationKind",
+          "updateCode": "(fun _ curr target -> triggerGoToAsync curr target)"
+        }
+      ]
+    },
+    {
+      "name": "Xamarin.Forms.ShellGroupItem",
+      "members": [
+        {
+          "name": "FlyoutDisplayOptions",
+          "defaultValue": "Xamarin.Forms.FlyoutDisplayOptions.AsSingleItem"
+        }
+      ]
+    },
+    {
+      "name": "Xamarin.Forms.SelectableItemsView",
+      "members": [
+        {
+          "name": "SelectedItem",
+          "defaultValue": "null"
+        },
+        {
+          "name": "SelectionChangedCommand",
+          "defaultValue": "null",
+          "inputType": "unit -> unit",
+          "convToModel": "makeCommand"
+        },
+        {
+          "name": "SelectionChangedCommandParameter",
+          "defaultValue": "null"
+        },
+        {
+          "name": "SelectionMode",
+          "uniqueName": "selectableItemsMode",
+          "defaultValue": "Xamarin.Forms.SelectionMode.None",
+          "inputType": "Xamarin.Forms.SelectionMode",
+          "modelType": "Xamarin.Forms.SelectionMode"
+        },
+        {
+          "name": "SelectionChanged",
+          "defaultValue": "null",
+          "inputType": "Xamarin.Forms.SelectionChangedEventArgs -> unit",
+          "convToModel": "(fun f -> System.EventHandler<Xamarin.Forms.SelectionChangedEventArgs>(fun _sender args -> f args))"
+        }
+      ]
+    },
+    {
+      "name": "Xamarin.Forms.ShellNavigationState",
+      "members": [
+        {
+          "name": "Location",
+          "defaultValue": "null",
+          "inputType": "System.Uri",
+          "convToModel": "makeUri"
+        }
+      ]
+    },
+    {
+      "name": "Xamarin.Forms.ShellContent",
+      "members": [
+        {
+          "name": "Content",
+          "defaultValue": "null",
+          "inputType": "ViewElement"
+        },
+        {
+          "name": "ContentTemplate",
+          "defaultValue": "null",
+          "inputType": "Xamarin.Forms.DataTemplate",
+          "convToModel": "makeTemplate"
+        }
+      ]
+    },
+    {
+      "name": "Xamarin.Forms.ShellItem",
+      "members": [
+        {
+          "name": "CurrentItem",
+          "defaultValue": "null"
+        }
+      ]
+    },
+    {
+      "name": "Xamarin.Forms.ShellSection",
+      "members": [
+        {
+          "name": "CurrentItem",
+          "defaultValue": "null"
+        },
+        {
+          "name": "GoToAsync",
+          "uniqueName": "ssGoToAsync",
+          "inputType": "string list * Map<string, string> * Fabulous.DynamicViews.AnimationKind",
+          "modelType": "string list * Map<string, string> * Fabulous.DynamicViews.AnimationKind",
+          "updateCode": "(fun _ curr target -> triggerSSGoToAsync curr target)"
+        }
+      ]
+    },
+    {
+      "name": "Xamarin.Forms.CarouselView",
```

Comment:
CarouselView and CollectionView have no members on their own?

Reply:
I just looked at fuget and there were no members. Maybe they will get some in 3.6? 
I check this.

Reply:
The GitHub classes of those two seems to be the same so it's correct?

Reply:
Ah ok. They don't have much code by themselves, it's mostly inherited from ItemsView/SelectableItemsView. Alright.